cif_pow.dic

```diff
@@ -3110,6 +3110,560 @@ save_pd_calib_wavelength.special_details
 
 save_
 
+save_PD_CALIB_XCOORD
+
+    _definition.id                PD_CALIB_XCOORD
+    _definition.scope             Category
+    _definition.class             Loop
+    _definition.update            2023-05-06
+    _description.text
+;
+    This section defines the parameters used for the calibration of the
+    X-coordinate axis of the instrument used directly or indirectly in the
+    interpretation of this data set; that is, 2\q, time-of-flight, or
+    position.
+
+    Calibration is carried out be adding the offset value:
+
+        X-coord~calibrated~ = X-coord~meas~ + X-coord~offset~
+
+    For cases where the offset value is not a constant, but rather varies
+    with X-coordinate, a set of offset values is supplied in a loop. In this
+    case, the value where the offset has been determined can be specified as
+    _pd_calib_xcoord.*_nominal. These values may need to be interpolated to
+    be applied to their respective diffractograms.
+
+    This category should be used to specify corrections relating to specimen
+    displacement, and other such specimen-related corrections.
```

Comment:
I think we should leave this sentence out. It seems to me that the basic specimen displacement parameters, where available, would be the more appropriate items to record.